OpenKeychain/src/main/java/org/sufficientlysecure/keychain/provider/KeychainDatabase.java

```diff
@@ -306,6 +314,11 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     return;
                 }
             */
+            case 20:
+                db.execSQL(CREATE_OVERRIDDEN_WARNINGS);
+                if (oldVersion == 20) {
+                    return;
+                }
```

Comment:
nice catch! we had the same problem with an upgrade from version 14 to anything newer! I guess we were lucky this never came up before